lib/unpack.js

```diff
@@ -13,7 +13,7 @@ var mkdirp = require("mkdirp");
 var path = require("path");
 
 function bowerOrPackageJson(entry) {
-  return /^(bower|package)\.json/.test(entry.entryName);
+  return /^((bower|package)\.json|.gitignore|README.md)/.test(entry.entryName);
```

Comment:
Now that this function handles additional files, what if we rename this function into something like `filesToIgnore`?